platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -119,9 +119,10 @@ android {
     }
 
     lintOptions {
+        disable 'MissingTranslation', 'TypographyQuotes'
```

Comment:
capturing from chat with @fabian-guerra that lint configuration will be restored once translations have landed on Transifex.